styles/colors-silicon-day.less

```diff
@@ -320,15 +321,23 @@
 
 // Switch
 // ---------------------------------------
-@agate-switch-bg-color: @agate-darker-gray;
-@agate-switch-selected-bg-color: @agate-accent;
-@agate-switch-selected-focus-bg-color: @agate-switch-selected-bg-color;
-@agate-switch-color: @agate-dark-gray;
-@agate-switch-selected-color: @agate-title-text-color;
+@agate-switch-bg-color: @agate-foreground;
+@agate-switch-selected-bg-color: @agate-foreground;
+@agate-switch-selected-focus-bg-color: @agate-foreground;
+@agate-switch-color: @agate-noble-gray;
+@agate-switch-selected-color: @agate-accent;
 @agate-switch-focus-bg-color: @agate-text-color;
 @agate-switch-focus-color: @agate-black;
 @agate-switch-label-color: @agate-accent;
 
+// SwitchItem
+// ---------------------------------------
+@agate-switchitem-label-color: fade(@agate-foreground, 30%);;
```

Comment:
Please adjust colours for Silicon day (white text) and remove the extra ";" at the end of this line.